src/test/simple_spin.cc

```diff
@@ -46,7 +46,8 @@ TEST(SimpleSpin, Test1)
   // Should also work with pass-by-reference:
   // (Note that we don't care about cross-threading here as-such.)
   counter = 0;
-  ASSERT_EQ(counter, 0);
+  uint32_t urhs = 0;
+  ASSERT_EQ(counter, urhs);
```

Comment:
The only point of the assertion after the assignment is to "prove" that the assignment precondition has indeed happened-- pedantic test-code paranoia (but, yes, it was intentional).